test/cypress/integration/field-extension.spec.js

```diff
@@ -0,0 +1,27 @@
+import { entry } from '../utils/paths'
+
+const post = {
+  id: '1MDrvtuLDk0PcxS5nCkugC',
+  title: 'My first post'
+}
+
+context('Field extension', () => {
+  beforeEach(() => {
+    cy.setAuthTokenToLocalStorage()
+  })
+
+  it('opens first post and checks that field extension is rendered', () => {
+    cy.visit(entry(post.id))
+
+    cy.getByText(post.title).should('exist')
+
+    // eslint-disable-next-line
+    cy.wait(3000)
+
+    cy.get('[data-field-api-name="title"] iframe').captureIFrameAs('extension')
```

Comment:
Abstract or at least add todo.

Reply:
I've abstracted the majority of the logic to `captureIFrameAs('extension')` already.
How do you see it?

Reply:
I was thinking about something even less selector-aware, like `getExtensionForField('title').find(...).assertion()` but let's keep it as it is right now.